source/analytics/templates/event.html.erb

```diff
@@ -0,0 +1,43 @@
+<h1 class="govuk-heading-l">
+  <span class="govuk-caption-l">Event</span>
+  <%= event.name %>
+</h1>
+
+<% if event.description %>
+  <p class="govuk-body"><%= event.description %></p>
+<% end %>
+
+<% event.events.each do |page_event| %>
+  <div class="event">
+    <h3 class="govuk-heading-m">
```

Comment:
The heading before this is a H1, so this should be a H2 (and the next one a H3, not a H4). Sorry, should have noticed this when I was pushing changes.